aws/service.go

```diff
@@ -482,6 +483,28 @@ func DatabaseMigrationService(ctx context.Context, d *plugin.QueryData) (*databa
 	return svc, nil
 }
 
+// DirectoryServiceService returns the service connection for AWS Directory service
+func DirectoryServiceService(ctx context.Context, d *plugin.QueryData) (*directoryservice.DirectoryService, error) {
```

Comment:
Function name can be `DirectoryService`, `Service`  word is getting repeated two times.